### ISSUE TITLE
Duplicate artifacts at maven central (FYI)

### ISSUE THREAD

**Issue by suraj-mathew** [tkohegyi/mitmProxy#25]
@tkohegyi, 

The maven central repository contains duplicate misleading artifact entries for this project, which seems to have got created during the initial phases of the project.

It will be great, if you can remove the old one from maven central to avoid confusion for future users.

- https://mvnrepository.com/artifact/website.magyar/mitmJavaProxy  (old one. to be deleted)
- https://mvnrepository.com/artifact/website.magyar/mitm-java-proxy  (new valid one) 

NOTE:
I didn't find any other means to convey this message to you. That's why raised this ticket.

**Comment by tkohegyi** [comment]
@suraj-mathew 
Thank you for contacting to me this way - it is perfect for this purpose.

Indeed, I know about this status and wanted to delete the old repo, just was unable to find the method - afaik, mvnrepository.com does not allow deleting. In case you know how to do it, pls let me know.

**Comment by suraj-mathew** [comment]
@tkohegyi  - That was a new piece of information to me, that mvn artifacts cant be deleted.. Hmm.. really strange... 
At least the owner account that pushes the binaries should have been given permission to remove... But, they might have their own rules...

When i searched some links in internet, one of them mentions to try the luck by dropping a detailed email to SonaType support via the email - [central-support@sonatype.com](mailto:central-support@sonatype.com)

This is a genuine reason, so you can give a try (if not done already).

**Comment by tkohegyi** [comment]
Hi @suraj-mathew - thank you for the suggestion, I will give a try -> mail sent to SonaType support to remove the unused repo.
We will see...

**Comment by tkohegyi** [comment]
Hi @suraj-mathew 

No luck  - quote from Sonatype response:

Sonatype policy prevents the deletion or any other modification of artifacts after they are published to Central. If you already have a newer version released, you could inform your users about avoiding the incorrect ones.

**Comment by suraj-mathew** [comment]
@tkohegyi  - :-( 

I think you can ask them on how the red boxed entry can be added to this mitm's old link ?

That way, even if someone access the old link - they will get notified to redirect.

Look at this sample: https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api

![Image](https://github.com/user-attachments/assets/69a34487-1cad-4c40-bc07-e5981d5e9b50)

**Comment by tkohegyi** [comment]
Yes, this is a helping KB - https://maven.apache.org/guides/mini/guide-relocation.html
Will do it as soon as I have time.
